src/content/changelog/4-62-0-2020-04-10.md

```diff
@@ -10,22 +10,22 @@ changelog:
 
 - The following Linode Kubernetes Engine (LKE) Beta Endpoints were added:
 
-  - List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](/api/v4/lke-clusters-cluster-id-api-endpoints)). This endpoint returns an array of Kubernetes API server endpoints for your LKE cluster.
+  - List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-api-endpoints-list)). This endpoint returns an array of Kubernetes API server endpoints for your LKE cluster.
 
 ### Changed
 
-- The View Kubernetes API Endpoint ([/lke/clusters/{clusterId}/api-endpoint](/api/v4/lke-clusters-cluster-id-api-endpoint)) endpoint is now deprecated in favor of List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](/api/v4/lke-clusters-cluster-id-api-endpoints)).
+- The View Kubernetes API Endpoint ([/lke/clusters/{clusterId}/api-endpoint](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-api-endpoints-list)) endpoint is now deprecated in favor of List Kubernetes API Endpoints ([GET /lke/clusters/{clusterId}/api-endpoints](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-api-endpoints-list)).
 
 - The `version` property of an LKE cluster has been updated to be named `k8s_version`. This change affects the following endpoints:
 
-    - List Kubernetes Clusters ([GET /lke/clusters](/api/v4/lke-clusters))
-    - Create Kubernetes Clusters ([POST /lke/clusters](/api/v4/lke-clusters/#post))
-    - View Kubernetes Cluster ([GET /lke/clusters/{clusterId}](/api/v4/lke-clusters-cluster-id))
+    - List Kubernetes Clusters ([GET /lke/clusters](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-clusters-list))
+    - Create Kubernetes Clusters ([POST /lke/clusters](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-cluster-create))
+    - View Kubernetes Cluster ([GET /lke/clusters/{clusterId}](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-cluster-view))
 
 ### Fixed
 
-- An [LKE cluster's](/api/v4/lke-clusters) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/api/v4/#filtering-and-sorting). These properties have been updated in the [Linode API v4 schema](/api/v4) to no longer display the `filterable` label.
+- An [LKE cluster's](https://www.linode.com/docs/api/linode-kubernetes-engine-lke/#kubernetes-clusters-list) `label`, `k8s_version`, and `region` properties were incorrectly labeled as being [filterable](/https://www.linode.com/docs/api/). These properties have been updated in the [Linode API v4 schema](https://www.linode.com/docs/api) to no longer display the `filterable` label.
```

Comment:
and here: http://www.linode.com/docs/api/#filtering-and-sorting

Reply:
pushed!